packages/datadog-plugin-rhea/test/index.spec.js

```diff
@@ -0,0 +1,544 @@
+
+            container.once('sendable', _context => {
+              context = _context
+              done()
+            })
+            const connection = container.connect({
+              username: 'admin',
+              password: 'admin',
+              host: 'localhost',
+              port: 5673
+            })
+            connection.open_sender('amq.topic')
+            connection.open_receiver('amq.topic')
+          })
+
+          describe('sending a message', () => {
+            it('should automatically instrument', (done) => {
+              agent.use(traces => {
+                const span = traces[0][0]
+                expect(span).to.include({
+                  name: 'rhea.sender.send',
+                  resource: 'amq.topic',
+                  error: 0,
+                  service: 'test-amqp'
+                })
+                expect(span.meta).to.include({
+                  'span.kind': 'producer',
+                  'amqp.link.target.address': 'amq.topic',
+                  'amqp.link.role': 'sender',
+                  'amqp.delivery.state': 'accepted',
+                  'out.host': 'localhost',
+                  'out.port': '5673'
+                })
+              })
+                .then(done, done)
+              context.sender.send({ body: 'Hello World!' })
+            })
+
+            it('should inject span context', () => {
+              container.on('message', msg => {
+                const keys = Object.keys(msg.message.delivery_annotations)
+                expect(keys).to.include('x-datadog-trace-id')
+                expect(keys).to.include('x-datadog-parent-id')
+              })
+            })
+          })
+
+          describe('receiving a message', () => {
+            it('should automatically instrument', done => {
+              agent.use(traces => {
+                const span = traces[0][0]
+                expect(span).to.include({
+                  name: 'rhea.receiver.onmessage',
+                  resource: 'amq.topic',
+                  error: 0,
+                  service: 'test-amqp'
+                })
+                expect(span.meta).to.include({
+                  'span.kind': 'consumer',
+                  'amqp.link.source.address': 'amq.topic',
+                  'amqp.link.role': 'receiver'
+                })
+              })
+                .then(done, done)
+              context.sender.send({ body: 'Hello World!' })
+            })
+
+            it('should extract the span context', done => {
+              container.on('message', msg => {
+                const span = tracer.scope().active()
+                expect(span._spanContext._parentId).to.not.be.null
+                done()
+              })
+              context.sender.send({ body: 'Hello World!' })
+            })
+          })
+
+          describe('exception in message handler', () => {
+            it('span should have error', (done) => {
```

Comment:
Nitpick, but chaining describes and its should read as a sentence in BDD, which is not the case here for "Plugin with broker without configuration exception in message handler span should have error". Similar instances also exist for other describes/its.